### PR TITLE
feat: fe-a-2021-q1-q30

### PR DESCRIPTION
**Pull request by jonathan-christ**
Added:
- Q1-Q30 flashcards of PhilNits FE April 2021 AM Questions
  - w/ corresponding pictures
- `#software` tag

### CODE REVIEW

**Comment by jonathan-christ** [comment]
These additional commits might be because of the frequent rebasing

**Review by Anjeelow — APPROVED** [review]

**Review by Luzefiru — CHANGES_REQUESTED** [review]
Delete the Various Complements plugin please.

File: .obsidian/plugins/various-complements/main.js
Comment (sethonne) [inline .obsidian/plugins/various-complements/main.js:1]: we deleted this plugin iirc

**Review by sethonne — CHANGES_REQUESTED** [review]

File: .obsidian/plugins/various-complements/main.js
Comment (jonathan-christ) [reply .obsidian/plugins/various-complements/main.js:1]: oks fixed

**Review by Luzefiru — APPROVED** [review]
LGTM.

**Review by sethonne — APPROVED** [review]
Noice LGTM after fixing the readme conflicts

**Review by sethonne — APPROVED** [review]
LGTM part 2

**Pull request merged**